engineapi/alembic/versions/b4257b10daaf_call_request_types_and_metatx_holders.py

```diff
@@ -0,0 +1,171 @@
+"""Call request types and Metatx holders
+
+Revision ID: b4257b10daaf
+Revises: dedd8a7d0624
+Create Date: 2023-08-02 18:28:14.724453
+
+"""
+import uuid
+
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = 'b4257b10daaf'
+down_revision = 'dedd8a7d0624'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+
+    # Blockchains
+    op.create_table('blockchains',
+    sa.Column('id', sa.UUID(), nullable=False),
+    sa.Column('name', sa.VARCHAR(length=128), nullable=False),
+    sa.Column('chain_id', sa.Integer(), nullable=False),
+    sa.Column('testnet', sa.Boolean(), nullable=False),
+    sa.PrimaryKeyConstraint('id', name=op.f('pk_blockchains')),
+    sa.UniqueConstraint('id', name=op.f('uq_blockchains_id'))
+    )
+
+    op.create_index(op.f('ix_blockchains_chain_id'), 'blockchains', ['chain_id'], unique=False)
+    op.create_index(op.f('ix_blockchains_name'), 'blockchains', ['name'], unique=True)
+    
+    op.add_column('registered_contracts', sa.Column('blockchain_id', sa.UUID(), nullable=True))
+    op.create_foreign_key(op.f('fk_registered_contracts_blockchain_id_blockchains'), 'registered_contracts', 'blockchains', ['blockchain_id'], ['id'], ondelete='CASCADE')
+
+    # Manual - Start
+    op.execute(f"INSERT INTO blockchains (id, name, chain_id, testnet) VALUES ('{str(uuid.uuid4())}', 'polygon', 137, FALSE);")
```

Comment:
Can we do a single `INSERT INTO blockchains (...) VALUES` and add all blockchains in a single statement?

Can we add all blockchains we currently support? Ethereum Mainnet, Gnosis chain, etc.?

Reply:
Added